packages/kbn-interpreter/src/public/batched_fetch.js

```diff
@@ -97,6 +97,7 @@ async function processBatch(kfetch, batch) {
 
     results.forEach(({ id, result }) => {
       const { future } = batch[id];
+
```

Comment:
Probably can just not commit this at all, as it's not changed other than white space.

Reply:
Fair, I'll rebase